### DISCUSSION TITLE
different results at every run

### DISCUSSION THREAD

**Discussion by linparkkin** [projectdiscovery/httpx#580] [Q&A]
Hi all, 
I'm getting different results every time I run httpx on the same list of domains (<100), do you have any advice to improve the reliability of the output?
I'm currently running a simple:

`httpx -l <file> -title -fr -silent`

Should I increase the number of retries? 

Thanks to everyone

**Comment by ehsandeep** [comment]
Hey @linparkkin, it would be hard to make any suggestion without knowing the root cause. Do you think it's possible to share the input list so we can replicate this behavior? otherwise, you can try using `-proxy` flag or `-v` or `-debug` to understand the different responses of the same host on multiple runs, which will help to understand the issue and make the required changes.

**Reply by linparkkin** [reply]
thanks for your quick reply, I run `-debug` on a host that returned different responses on multiple runs, the error seems to be due to the default 5 seconds timeout:  `...context deadline exceeded (Client.Timeout exceeded while awaiting headers)`. By increasing it up to 10 with `-timeout` flag the results are more consistent, even if they are not the same 100% of the runs (which is understandable).
